### DISCUSSION TITLE
blog_11ty/posts/%E5%A6%82%E4%BD%95%E5%B0%86%E5%90%ABtext%E7%9A%84svg%E5%9B%BE%E8%BD%AC%E6%8D%A2%E4%B8%BApathSvg%EF%BC%88svg%E8%BD%AC%E6%9B%B2%EF%BC%89/

### DISCUSSION THREAD

**Discussion by giscus** [joisun/blog_11ty#5] [General]
# blog_11ty/posts/%E5%A6%82%E4%BD%95%E5%B0%86%E5%90%ABtext%E7%9A%84svg%E5%9B%BE%E8%BD%AC%E6%8D%A2%E4%B8%BApathSvg%EF%BC%88svg%E8%BD%AC%E6%9B%B2%EF%BC%89/

simple style blog site

https://jaycethanks.github.io/blog_11ty/posts/%E5%A6%82%E4%BD%95%E5%B0%86%E5%90%ABtext%E7%9A%84svg%E5%9B%BE%E8%BD%AC%E6%8D%A2%E4%B8%BApathSvg%EF%BC%88svg%E8%BD%AC%E6%9B%B2%EF%BC%89/

**Comment by joisun** [comment]
bravo~
